modules/hub/package.json

```diff
@@ -64,18 +65,15 @@
     "handy-redis": "^1.3.0",
     "human-standard-token-abi": "^1.0.2",
     "logdna": "^2.0.2",
-    "machinomy": "https://github.com/mslipper/machinomy#a11952271329d795d4b7c7d9de959681a7dc3a52",
     "nock": "^10.0.1",
     "node-fetch": "^2.0.0",
-    "node-sass-chokidar": "^0.0.3",
     "nodemon": "^1.18.9",
     "path-to-regexp": "^2.1.0",
     "pg": "^7.4.1",
     "pg-types": "^1.13.0",
     "sql-template-strings": "^2.2.2",
     "uuid": "^3.2.1",
-    "web3": "1.0.0-beta.34",
-    "web3-old": "npm:web3@^0.20.1",
```

Comment:
npm doesn't understand this syntax, not sure what this was supposed to do but it doesn't seem to be necessary.